client/my-sites/checkout/composite-checkout/components/checkout-terms-item.tsx

```diff
@@ -0,0 +1,53 @@
+import { Gridicon } from '@automattic/components';
+import styled from '@emotion/styled';
+import classNames from 'classnames';
+
+const Wrapper = styled.div`
+	padding-left: 24px;
+	position: relative;
+	font-size: 12px;
+
+	> svg {
+		position: absolute;
+		top: 0;
+		left: 0;
+		width: 16px;
+		height: 16px;
+
+		.rtl & {
+			left: auto;
+			right: 0;
+		}
+	}
+
+	p {
+		font-size: 12px;
+		margin: 0;
+		word-break: break-word;
+	}
+`;
+
+const CheckoutTermsItem = ( {
+	children,
+	className,
+	onClick,
```

Comment:
I think we should define a default value, i.e. a `noop` function.

Reply:
Or maybe not after all. I thought this would trigger an error if I clicked the terms where that even handler wouldn't be provided but I didn't notice anything in my browser's console.

Reply:
We should be fine without a noop default! React accepts `undefined` or a function as the `onClick` prop to a `div`. But just for the sake of it – I updated the type for `onClick` to be a little bit more strict.